hudi-utilities/src/main/java/org/apache/hudi/utilities/HiveIncrementalPuller.java

```diff
@@ -112,7 +112,7 @@ public HiveIncrementalPuller(Config config) throws IOException {
     this.config = config;
     validateConfig(config);
     String templateContent =
-        FileIOUtils.readAsUTFString(this.getClass().getResourceAsStream("IncrementalPull.sqltemplate"));
+        FileIOUtils.readAsUTFString(this.getClass().getResourceAsStream("/IncrementalPull.sqltemplate"));
```

Comment:
rebase master branch.

Reply:
hi @hmatu, Very sorry. I'm not familiar with submitting PR for the first time. Redundant code has been submitted, Can you help delete this pull, I will create a new PR